node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -463,6 +463,13 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       descr = "the number of the active validators"
     )
 
+    val posVaultPubKey = opt[String](
+      descr = "Public key for transfers from the PoS vault. " +
+        "Base16 encoding of the public key to use for signing a proposed blocks. Can be inferred from the" +
```

Comment:
I think this is incorrect. This key is not used to sign any block. Better to say `Public key for transfers from the PoS vault. Base16 encoding of the public key of PoS vault owner.`  @zsluedem do you agree?

Reply:
You are right~! @maximus19910112 Please rephrase.